namespaces-disallowlist.yaml

```
@@ -11,3 +11,4 @@
 - pioneer_*
 - rally_*
 - reference_browser
+- org_mozilla_tiktokreporter
```

```suggestion
- tiktokreporter_*
- org_mozilla_ios_tiktok_reporter*
- org_mozilla_tiktokreporter
```
These two additional lines might be needed, but we can try without them.